CONTRIBUTING.md

```diff
@@ -80,7 +80,12 @@ see [RubyMine IDE Debugging](#rubymine-ide-debugging) or [Visual Studio Code IDE
 The `dev` directory contains a `docker-compose` file which creates a development
 environment with a database container (`pg`, short for *postgres*), and a
 `conjur` server container with source code mounted into the directory
-`/src/conjur-server`.
+`/src/conjur-server`. `conjur` listens on `127.0.0.1:3000`.
+
+Also, there's a `proxy` container that wraps the conjur server with HTTPS
+redirection using nginx. `proxy` listens on `127.0.0.1:8443`.
```

Comment:
so I assume all requests should be made to this IP and port to communicate with Conjur over TLS?